modules/src/core/ics02_client/msgs/upgrade_client.rs

```diff
@@ -100,9 +100,9 @@ impl TryFrom<RawMsgUpgradeClient> for MsgUpgradeAnyClient {
             client_state: AnyClientState::try_from(raw_client_state)?,
             consensus_state: AnyConsensusState::try_from(raw_consensus_state)?,
             proof_upgrade_client: RawMerkleProof::try_from(c_bytes)
-                .map_err(Error::invalid_upgrade_client_proof)?,
+                .map_err(Error::invalid_client_proof)?,
```

Comment:
It's unclear why we changed these types here. It seems like the types were more accurate in the original code.

Reply:
I think this was to reuse these errors in the verification functions. But as you rightly pointed out that those errors are better expressed as a separate variant, I will revert the names of these errors. 👍